Dockerfile-bundle-base

```diff
@@ -9,6 +9,6 @@ RUN set -ex && \
   chmod g+w $HOME && \
   apk add --update --no-cache make gcc libc-dev
 
-RUN gem install bundler -v 1.17.3
+RUN gem install bundler -v 2.4.0
```

Comment:
If you do `gem install bundler -v "~> 2.4"` then it will allow non breaking changes to the version.